libs/auth-api-lib/src/lib/personal-representative/models/personal-representative-delegation-type.model.ts

```diff
@@ -0,0 +1,74 @@
+import {
+  BelongsTo,
+  Column,
+  CreatedAt,
+  DataType,
+  ForeignKey,
+  Model,
+  PrimaryKey,
+  Table,
+  UpdatedAt,
+} from 'sequelize-typescript'
+import { ApiProperty } from '@nestjs/swagger'
+import { PersonalRepresentative } from './personal-representative.model'
+import { DelegationTypeModel } from '../../delegations/models/delegation-type.model'
+
+@Table({
+  tableName: 'personal_representative_delegation_type',
+  indexes: [
+    {
+      fields: ['personal_representative_id'],
+    },
+    {
+      fields: ['delegation_type_id'],
+    },
+  ],
+})
+export class PersonalRepresentativeDelegationTypeModel extends Model {
```

Comment:
Add type-safety definition to the model definition using the `InferAttribtues` and `InferCreationAttributes`.